bitcoin/Cargo.toml

```diff
@@ -36,7 +36,8 @@ units = { package = "bitcoin-units", version = "0.1.0", default-features = false
 
 base64 = { version = "0.22.0", optional = true }
 ordered = { version = "0.2.0", optional = true }
-bitcoinconsensus = { version = "0.106.0+26", default-features = false, optional = true }
+# Check cargo tree to see the real version used (and thus the Core version) as you would normally with any other dependency
```

Comment:
```suggestion
# `bitcoinconsensus` version includes a build number which indicates the version of Core, use cargo tree to see it. 
```

Reply:
That's not a build number, that's version metadata.

Reply:
> Tobin's suggested text sounds good except for the run-on sentence. His , should be a ..

Agree, but I think semicolon would read better.